frontend/src/metabase/routes.jsx

```diff
@@ -146,6 +147,19 @@ export const getRoutes = store => {
             component={TrashCollectionLanding}
           />
 
+          <Route
+            path="collection/by-entity-id/:slug(**)"
```

Comment:
`(**)` matches also all sub-paths, so that we support `/move` `/archive` etc out